cli/resources/workspace/mvnet-allow-only-approved-outbound.yml

```diff
@@ -0,0 +1,19 @@
+name: myworkspace_dep
+managed_network:
+  isolation_mode: allow_only_approved_outbound
+  outbound_rules:
+  - name: added-servicetagrule
```

Comment:
they can add outbound rules with IP prefixes too. Please add that into this example

Reply:
no, this is not supported. we only support PE, ServiceTag, FQDN. not sure what you mean by this

Reply:
ip address outbound rule does not seems supported from service, also do not think this is included in SDK/CLI either yet? will not include it in this example. let me know if I am missing something here